en/docs/administer/carbon-secure-vault-implementation.md

```diff
@@ -205,14 +196,11 @@ To create a custom secret repository, you need to implement the
 
 3.  Create a JAR or an OSGI bundle.
 
-4.  Then, copy the JAR file to the
-    `               <PRODUCT_HOME>/repository/component/lib/              `
-    directory or the OSGI bundle to the
-    `               <PRODUCT_HOME>/repository/component/              `
-    dropins `               /              ` directory .
+4.  Then, copy the JAR file to the `               <PRODUCT_HOME>/repository/component/lib/              ` directory or the OSGI bundle to the
+    `               <PRODUCT_HOME>/repository/component/              ` dropins `               /              ` directory .
```

Comment:
shall we change the <PRODUCT_HOME> to <IS_HOME> wherever applicable now that we have moved everything to the IS space?